doc/standards/testStrategy.md

```diff
@@ -0,0 +1,119 @@
+# Testing Startegy
+
+## Why
+
+- Code consistency
+- Tests readability
+
+## Key points
+
+- **Full page test**
+
+```
+it('should render correctly without login modal', async () => {
+   const home = await homeRenderer(false)
+
+   expect(home).toMatchSnapshot()
+   home.unmount()
+})
+```
+
+You should mock SVG & Lottie animations so snapshot can be more lisible
+
+- **Graphics Component test**
+
+- ❌ No snapshots tests for component. But you can use diff snapshot for complex style checking
```

Comment:
Tu peux supprimer le tiret pour que ça fasse un item au même niveau que le titre :)
Et je rajouterai juste un peu de why, genre : 
"Why pages' snapshots and not components ones:
If I modify a component > component snapshot would for sure be modified : useless What I want to know when I modify a component is which pages are impacted > pages snapshots useful "